### ISSUE TITLE
[Bug]: [benchmark][standalone]Milvus interface response times are getting slower

### ISSUE THREAD

**Issue by jingkl** [milvus-io/milvus#16041]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:2.0.2-20220309-7734b9e6
- Deployment mode(standalone or cluster):standalone
- SDK version(e.g. pymilvus v2.0.0rc2):
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

server-instance
benchmark-tag-no-clean-s44mh-1
server-configmap
server-single-32c128m
client-configmap
client-random-locust-100m-ddl-r8-w2

1. Milvus 2.0.2 query average response time(ms)：
<img width="1364" alt="1" src="https://user-images.githubusercontent.com/34296482/158141860-4c091932-399f-4d86-affe-19c69e1098e9.png">

Milvus 2.0 query average response time(ms)：
![3](https://user-images.githubusercontent.com/34296482/158144885-bb05fa11-1f34-4f44-913e-2428cb587726.png)

Milvus 2.0.2 search average response time(ms)：
![2](https://user-images.githubusercontent.com/34296482/158142815-7557746a-05b1-4fc2-b843-ebac3a85b46b.png)
Milvus 2.0 search average response time(ms)：
![4](https://user-images.githubusercontent.com/34296482/158144640-58114cf3-33f5-4d67-bcec-620f8bb371fa.png)

server：
```
benchmark-tag-no-clean-s44mh-1-etcd-0                             1/1     Running     0          4d     10.97.16.172   qa-node013.zilliz.local   <none>           <none>
benchmark-tag-no-clean-s44mh-1-milvus-standalone-6f47494bdp6pkx   1/1     Running     1          4d     10.97.17.146   qa-node014.zilliz.local   <none>           <none>
benchmark-tag-no-clean-s44mh-1-minio-76f5fcbf87-757zm             1/1     Running     0          4d     10.97.16.176   qa-node013.zilliz.local   <none>           <none>
```

### Expected Behavior

Milvus average response time should be increasing slowly

### Steps To Reproduce

```markdown
1、create collection
2、create index of ivf_sq8
3、insert 100m million vectors
4、flush collection
5、build index with the same params
6、load collection
7、locust concurrent: query<-search, load, get<-query, scene_test
```

### Anything else?

client-random-locust-100m-ddl-r8-w2
```
{
	"config.yaml": "locust_random_performance:
		  collections:
		    -
		      collection_name: sift_100m_128_l2
		      ni_per: 50000
		      build_index: true
		      index_type: ivf_sq8
		      index_param:
		        nlist: 2048
		      task:
		        types:
		          -
		            type: query
		            weight: 8
		            params:
		              top_k: 10
		              nq: 10
		              search_param:
		                nprobe: 16
		          -
		            type: load
		            weight: 1
		          -
		            type: get
		            weight: 8
		            params:
		              ids_length: 10
		          -
		            type: scene_test
		            weight: 2
		        connection_num: 1
		        clients_num: 20
		        spawn_rate: 2
		        during_time: 302400
		"
}
```

**Comment by yanliang567** [comment]
It looks like a performance regression issue since 2.0, please fix it in 2.0.2 if possible. @xiaofan-luan 

/assign @xiaofan-luan 
/unassign

**Comment by xiaofan-luan** [comment]
/assign @bigsheeper 
see if the monitoring metrics can help us on locating the performance degrade issue?

**Comment by bigsheeper** [comment]
> /assign @bigsheeper see if the monitoring metrics can help us on locating the performance degrade issue?

I'll take a look

**Comment by xiaofan-luan** [comment]
@bigsheeper any progress?

**Comment by wangting0128** [comment]
pymilvus==2.0.2.dev5

**Comment by jingkl** [comment]
argo
benchmark-tag-no-clean-r6zxf-1
server-configmap
server-single-32c128m
client-configmap
client-random-locust-50m-ddl-r8-w2-new

This is the result of monitoring the interface for two hours of operation：
Milvus 2.0.2  query average response time(ms)：
<img width="1359" alt="截屏2022-03-25 19 49 07" src="https://user-images.githubusercontent.com/34296482/160115655-04cfc799-7184-4197-91b1-9554efa1c719.png">
Milvus 2.0.2 search average response time(ms)：
<img width="1364" alt="截屏2022-03-25 19 49 37" src="https://user-images.githubusercontent.com/34296482/160115681-e5c539ea-8e4a-4da1-89cc-a360973cf785.png">

server:
```
MINATED NODE   READINESS GATES
benchmark-tag-no-clean-r6zxf-1-etcd-0                             1/1     Running     0          93s   10.97.16.252   qa-node013.zilliz.local   <n
one>           <none>
benchmark-tag-no-clean-r6zxf-1-milvus-standalone-56d779dbbvb5v4   1/1     Running     0          93s   10.97.20.47    qa-node018.zilliz.local   <n
one>           <none>
benchmark-tag-no-clean-r6zxf-1-minio-5887b557d8-4vzkw             1/1     Running     0          93s   10.97.12.147   qa-node015.zilliz.local   <n
one>           <none>
```


- Milvus version:2.0.2-20220324-8f4479a4
- Deployment mode(standalone or cluster):standalone
- SDK version(e.g. pymilvus v2.0.0rc2):2.0.2dev9

**Comment by xiaofan-luan** [comment]
could be related to #16196

**Comment by bigsheeper** [comment]
could be fixed by #16216

**Comment by bigsheeper** [comment]
should be fixed by #16287 , please verify @jingkl

**Comment by jingkl** [comment]
milvus ：master-20220519-599763d9
pymilvus：pymilvus-2.1.0.dev39
<img width="1342" alt="截屏2022-05-20 13 45 05" src="https://user-images.githubusercontent.com/34296482/169459092-139207f2-59e6-4fad-b6e0-9b121f4071b6.png">
Already fixed, turn off issue
